modules/dynatrace-activegate/variables.tf

```diff
@@ -30,10 +30,18 @@ variable "dynatrace_instance_name" {
   default = "ebe20728"
 }
 
+variable "dynatrace_nonprod_instance_name" {
+  default = "yrk32651"
+}
+
 variable "instance_count" {
   default = 1
 }
 
+variable "instance_count_nonprod" {
+  default = 2
```

Comment:
this should default to 1 so as testing of this module doesn't deploy more than needed,
it should pass in 2 to the module when it's used:

https://github.com/hmcts/azure-platform-terraform/blob/master/components/dynatrace_activegate/main.tf

Also the original design was for all environments to go through the same active gate, since a nonprod tenant is being introduced then prod should probably be downscaled to 2?

Reply:
agree with Tim, leave default at 1 and pass in 2.

If it were me I'd have everything go through the 1 set of activegates but it sounds like the decisions already been made

Reply:
It has been agreed that non-prod will have two activegates and prod will keep its original 3. Will get a review when the code is ready for the moment PR will be closed